app/presenters/metrics_presenter.rb

```diff
@@ -36,6 +37,7 @@ def metric_groups
       metric_groups.reverse! if order == Metrics::Order::Descending
 
       if order_by == Metrics::OrderBy::Name.identifier
+        metric_groups = metric_groups.reverse
```

Comment:
Ah, yeah. I'm put off by the fact that we can potentially reverse the list twice, and ignoring performance completely (I don't think that matters here), I worry that it creates confusion. If I'm reading the code is it clear why this is being reversed again? Is there a descriptive name we can use to replace the `order == Metrics::Order::Descending` condition above?

Having said that, this is relatively minor, so I'm happy for this to be merged.